common-pg/src/main/scala/com/socrata/pg/store/SecondarySchemaLoader.scala

```diff
@@ -103,6 +113,34 @@ class SecondarySchemaLoader[CT, CV](conn: Connection, dsLogger: Logger[CT, CV],
     }
   }
 
+  /**
+    * To suppress index create
+    *   INSERT into index_directives(dataset_system_id, field_name, directives) VALUES (${datasetSystemId}, ${fieldName}, '{"enable": false}')
+    *      (fieldName can be null to apply to all columns)
+    * To trigger reindex
+    *   curl -v  -X POST -H 'Content-Type: application/json' 'http://localhost:6010/dataset/${resourceName}/!secondary-reindex' --data '{}'
+    */
+  private def shouldCreateIndex(stmt: PreparedStatement, datasetId: DatasetId, fieldName: Option[ColumnName]): Boolean = {
```

Comment:
Hmm.  If I recall correctly the fieldName was optional for migration from a world where field names were not plumbed all the way down to a world where it was.  It might be a good idea to deoptionalize that at some point.  I was confused for a while why this was Optional

Reply:
yep, we can do the clean up next